api/swagger.yaml

```diff
@@ -449,15 +449,14 @@ definitions:
               should be provided as as 2-length arrays, where the first item is
               the key and the second the value.
             type: "array"
-            properties:
+            items:
+              type: "array"
               items:
                 type: "array"
```

Comment:
That looks like an `[][][]string` instead of `[][]string`?

Reply:
🤦 that's why I shouldn't do late-night coding 🙈 

Should be fixed now 👍